draft-gpew-priv-ppm.md

```diff
@@ -810,29 +810,42 @@ PPM task, the leader issues an "aggregate-share request" to each helper. The
 helper responds to this request by extracting its aggregate share from its state
 and encrypting it under the collector's HPKE public key.
 
-For each aggregator endpoint `[aggregator]` in the parameters associated with
-`CollectReq.task_id` (see {{pa-collect}}) except its own, the leader sends a
-POST request to `[aggregator]/aggregate_share` with the following message:
+First, the leader computes a checksum over the set of output shares included in
+the batch window. The checksum is computed by taking the SHA256 of the nonces
+for the client reports included in the aggregation, then combining the hash
+values with a bytewise-XOR operation.
```

Comment:
nit:
```suggestion
values with a bitwise-XOR operation.
```

Reply:
Huh, I always thought "bytewise" was more common than "bitwise", but a quick Googling shows this is not the case.